### DISCUSSION TITLE
ActionSheet Download Button How-To

### DISCUSSION THREAD

**Discussion by cproinger** [UI5/webcomponents-react#4900] [Q&A] [answered]
i want to provide a File-Download via an ActionSheet. Since ActionSheet does stuff to it's children (wrapping it in buttons?) i cannot get this to work with such a markup
```
<a href={data.links?.downloadURL} download={data.description}>
      <Button icon={'download'}>Download</Button>
</a>
```
any suggestions on how to implement this?

**Comment by MarcusNotheis** [comment, marked as answer]
Hi @cproinger,

unfortunately it is not possible to achieve this behaviour using an `ActionSheet`. According to the Fiori Guidelines, only `Button` components should be used inside an `ActionSheet`.

In order to achieve your desired markup, I would recommend to implement a `Popover` or `ResponsivePopover` on your own and place the download link button in there.
